server/backend/habit.py

```diff
@@ -160,7 +160,7 @@ def frequency(self, frequency: int):
         if not isinstance(frequency, int):
             raise Exception("frequency must be an int")
         if frequency < CompletionFrequency.DAILY.value or frequency > CompletionFrequency.MONTHLY.value:
-            raise Exception("frequency must be betwwn 0 and 2, inclusive")
+            raise Exception("frequency must be beteen 0 and 2, inclusive")
```

Comment:
This is what I am referring to.

Reply:
Spelling mistake got corrected, gonna close this convo.